Languages/DSA/questions.json

```diff
@@ -98,6 +98,33 @@
       "trueOrFalse": false,
       "shortAns": "It is an efficient algorithm to search an element in a sorted list",
       "tag": "binary-search"
+    },
+    {
+      "id": 11,
+      "question": "Minimum number of queues required for priority queue implementation?",
+      "details": "Minimum number of queues required for priority queue implementation is two. One for storing actual data and one for storing priorities.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "Atleast two queues requried for priority queue implementation.",
+      "tag": "queues"
+    },
+     {
+      "id": 12,
+      "question": "What is a graph?",
+      "details": "A graph is a non-linear data structure, which consists of vertices (or nodes) connected by edges (or arcs) where edges may be directed or undirected. In Computer science graphs are used to represent the flow of computation..",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "A graph is a pictorial representation of a set of objects where some pairs of objects are connected by links. The interconnected objects are represented by points termed as vertices, and the links that connect the vertices are called edges.",
```

Comment:
Again, please dont do "What is a cat" and then " A cat is" short answers should not use the same terminology as used in the question.

Reply:
Now here do i need to change in both detail answer and short answer?
I need to start the answer from "It is" but not with "The graph is"...

Reply:
Only need to change the shortAns, look at the Android json files for reference.